operations/jsonnet/microservices/config.libsonnet

```
@@ -18,6 +18,9 @@
     node_selector: null,
     ingester_allow_multiple_replicas_on_same_node: false,
 
+    // disable tempo-query by default
+    tempo_query: false,
```

```suggestion
    tempo_query: {
        enabled: false,
    },
```

tempo-query has some TLS config options that should be configurable. This way, it opens the possibility of adding them.

---

makes sense, turned it into an object.